pkg/kor/deployments.go

```diff
@@ -83,3 +84,16 @@ func GetUnusedDeploymentsJSON(includeExcludeLists IncludeExcludeLists, kubeconfi
 
 	return string(jsonResponse), nil
 }
+
+func GetUnusedDeploymentsYAML(includeExcludeLists IncludeExcludeLists, kubeconfig string) (string, error) {
```

Comment:
This can exist within the `GetUnusedDeploymentsJson` function.
It can be renamed to `GetUnusedDeploymentsStructured`